src/file-operations.js

```diff
@@ -17,9 +17,15 @@ function readFile(file, readDirPath) {
 function parseFile(fileData, { templateName, packageName, app, rnVersion }) {
   let kotlinPackage;
   let javaPackage;
-
+  var version = rnVersion;
+  debugger;
```

Comment:
@dlowder-salesforce this should be removed

Reply:
oops sorry